### PR TITLE
June 2021 Changelog

### PR DESCRIPTION
**Pull request by carolyn-shannon**
**Closes:** #6408 

## Summary

**[June 2021 Changelog](https://pantheon.io/docs/changelog)** -  Pantheon Changelog for June 2021.

## Post Launch

**Do not remove** - To be completed by the docs team upon merge:

- [ ] Redirect `/docs/old-path/` => `/docs/new-path/` (if applicable)
- [ ] Include/exclude pages ^ respectively within docs search service provider (if applicable)
- [ ] For Heroes - add a props post to the [discussion board](https://discuss.pantheon.io/c/pantheon-platform/documentation/17).
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | [Live](https://pantheon.io/docs) | [pr-6434](https://pr-6434--pantheon-docs.my.pantheonfrontend.website/docs) | Difference |
| --- | --- | --- | --- |
| Performance Score | `53` | `69` | `16` |
| Best Practice Score | `79` | `79` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `96` | `96` | `0` |

Lighthouse Audit Reports:

- [`pr-6434` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/27592/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/27592/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)


[Preview Environment](https://pr-6434--pantheon-docs.my.pantheonfrontend.website/docs)

**Comment by pwtyler** [comment]
### WordPress 5.7.2
WordPress was updated to [5.7.2](https://wordpress.org/news/2021/05/wordpress-5-7-2-security-release/).

File: source/changelogs/2021-06-01-June.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+### WordPress 5.7.2
+
+[WordPress 5.7.2](https://wordpress.org/news/2021/05/wordpress-5-7-2-security-release/) is now available on the Pantheon platform. This release fixes security vulnerabilities, and users are urged to upgrade their sites immediately. Detailed information on applying and debugging core updates can be found in the [Core Updates documentation](/core-updates) documentation. For more information, see the [WordPress 5.7.2](https://wordpress.org/news/2021/05/wordpress-5-7-2-security-release/) release notes.
+
+
+## Sponsored Projects
+
+### WP-CLI 2.5.0
+
+[WP-CLI 2.5.0](https://make.wordpress.org/cli/2021/05/19/wp-cli-v2-5-0-release-notes/) is now available. This release fixes security vulnerabilities, and users are urged to upgrade their sites. For more information, see the [WP-CLI 2.5.0](https://make.wordpress.org/cli/2021/05/19/wp-cli-v2-5-0-release-notes/) release notes. Detailed information on using WP-CLI on the Pantheon Platform can be found in the [WP-CLI](/wp-cli) documentation.
```
Comment (pwtyler) [inline source/changelogs/2021-06-01-June.md:23]: Composer-driven WordPress users may be in control of WP-CLI versions (I know this is possible with Drush), but WP-CLI version is usually controlled by the platform, and most users won't need to make any changes, 2.5.0 is already available.

**Review by joa-pan — APPROVED** [review]
LGTM

**Pull request merged**